parts/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -36,10 +36,16 @@ spec:
           mountPath: "/etc/kubernetes"
         - name: "var-lib-kubelet"
           mountPath: "/var/lib/kubelet"
+        - name: "msi"
+          mountPath: "/var/lib/waagent/ManagedIdentity-Settings"
+          readOnly: true
   volumes:
     - name: "etc-kubernetes"
       hostPath:
         path: "/etc/kubernetes"
     - name: "var-lib-kubelet"
       hostPath:
         path: "/var/lib/kubelet"
+    - name: msi
+      hostPath:
+      path: "/var/lib/waagent/ManagedIdentity-Settings"
```

Comment:
1. It's used for SSH key propagation (it's a GKE thing I think). apiserver also parses the config file and passes it to admission controllers so they can use it. (Not sure of any that do).
2. We don't use SSH key stuff and we don't use any admission controllers that need to talk to ARM so it didn't hit on this.

Still, will fix, of course.